packages/ckeditor5-ui/src/editorui/boxed/boxededitoruiview.js

```diff
@@ -43,6 +44,13 @@ export default class BoxedEditorUIView extends EditorUIView {
 		 */
 		this.main = this.createCollection();
 
+		/**
+		 * Editor's title based on the passed object options.editorName.
+		 *
+		 * @readonly
+		 */
+		this.label = options ? options.editorName : '';
```

Comment:
*   `BoxedEditorUIView` tests missing for this logic.
*   Also, does this thing need to be public? I see no good reason to expose it yet.
*   Also, let's name it `_editorName` maybe (?). No reason to rename it here, the option is pretty clear as to what information it carries.